src/js/gi/reducers/index.js

```diff
@@ -9,7 +11,7 @@ import InstitutionDetailsFilterReducer from './filter';
 import ProfileReducer from './profile';
 import CalculatorReducer from './calculator';
 
-const rootReducer = {
+const rootReducer = combineReducers({
```

Comment:
In which case, probably using `createCommonStore` again in the tests would avoid the verbosity of using `createStore` and `combineReducers` in each of them.